packages/oas-utils/src/spec-getters/getRequestBodyFromOperation.ts

```diff
@@ -45,6 +45,7 @@ const mimeTypes = [
 export function getRequestBodyFromOperation(
   operation: Omit<TransformedOperation, 'httpVerb'>,
   selectedExampleKey?: string | number,
+  omitEmptyAndOptionalProperties?: boolean,
```

Comment:
~when making changes like this, its good to default the value to what it was before to prevent any unforeseen bugs.~
```suggestion
  omitEmptyAndOptionalProperties = true,
```

Reply:
but maybe what we had before was bad, I'm not sure of the context exactly, in that case nevermind!